api/src/main/kotlin/br/com/gamemods/minecity/api/claim/TrustLevel.kt

```diff
@@ -1,7 +1,7 @@
 package br.com.gamemods.minecity.api.claim
 
+import br.com.gamemods.minecity.api.id.ClaimPermissionId
 import br.com.gamemods.minecity.api.id.ClamFlagId
```

Comment:
Oops, small typo here. You fixed ClaimPermission, but we also have another typo: "ClamFlagId"